clientd3d/toolbar.c

```diff
@@ -24,6 +24,8 @@
 #include "client.h"
 
 #define MAX_BUTTONS 30
+// Define cooldown period in milliseconds (e.g., 1000 ms = 1 second)
+#define BUTTON_TOGGLE_COOLDOWN_PERIOD 250
```

Comment:
The average person appears to have a reaction time of around 200 ms. It's certainly possible to click faster. I tried to tune this to prevent your average double-click from causing the issue, while also not impeding everyday situations like resting and then quickly realizing you need to stand to run off somewhere.